apps/src/templates/certificates/petition/PetitionForm.jsx

```diff
@@ -87,25 +136,29 @@ const PetitionForm = () => {
         </ControlledFieldGroup>
         <ControlledFieldGroup
           id="profession"
+          name="role_s"
           placeholderOrLabel={i18n.iAmA()}
-          isErrored={invalidFields.includes('profession')}
+          isErrored={invalidFields.includes('role_s')}
           componentClass="select"
           onChange={handleChange}
-          value={data['profession'] || ''}
+          value={data.role_s || ''}
         >
           {[
-            '-',
-            i18n.student(),
-            i18n.parent(),
-            i18n.educator(),
-            i18n.administrator(),
-            i18n.softwareEngineer(),
-            i18n.noneOfTheAbove()
-          ].map((profession, index) => (
-            <option key={index} value={profession}>
-              {profession}
-            </option>
-          ))}
+            'placeholder',
```

Comment:
maybe `Object.values(professionsOptions).map(option => option.dataString)`

Reply:
I originally had that and then realized I can't depend on the ordering of the object (although upon inspection, it looks like the implementation across browsers is pretty consistent for object iteration). As it is, I think this will throw a TypeError if something is in the array but not in the options object –– thoughts?

Reply:
oo, good point! really, i'm just seeing if there is a nice way to make sure those strings aren't duplicated.. maybe make professionOptions an array and `professionOptions.find(option => option.dataValue === value)` (for such a small list, it doesn't matter too much whether its a dictionary or a search lookup)

Reply:
oh better yet, have an array of the dataValues and a map of dataValue to i18n string

Reply:
Or just a map?

Reply:
Map should preserve order. I just didn't see it much in our repo so was nervous lol

Reply:
I went down the Map route just now, but then would need to create a custom way of figuring out what the user selected for

```
// ensure profession data is sent in english
      sanitizedData.role_s =
        find(professionOptions, {text: data.role_s})?.dataString || 'other';
```

Reply:
Personally, I don't mind having the array in the code because it's clear what the options are without going anywhere else––and if there is a mismatch, an error will be thrown

Reply:
yeah, i'm just worried about when something needs to get added to that list, it'll be more work to maintain if its not a single source of truth. it's not a huge issue, though, whatever method works you find works best

Reply:
I finally took your advice! Thanks for your patience on this :)